main/zoe/guide/proposal.md

```diff
@@ -1,41 +1,68 @@
-# The Structure of Proposals
+# The Structure of Offers
 
 <Zoe-Version/>
 
-## Making an offer
+## Making An offer
 
-In order to make an offer, you must provide two things: the payments
-that will be escrowed as part of the offer, and a statement of the
-conditions of the offer. This statement, which is approved by the user
-in their wallet, is known as the proposal.
+To make an offer, you use [`E(zoe).offer()`](https://github.com/Agoric/documentation/blob/Alpha-Zoe-Objects/main/zoe/api/zoe.md#ezoeofferinvitation-proposal-paymentkeywordrecord), which takes three arguments:
+- An `invitation` to participate in this contract instance.
+- A `proposal` stating your offer conditions.
+- The `payments` escrowed for the offer, each in association with a `proposal`-specified keyword.
 
-## Structure of the proposal
+## Invitations
+
+`Invitations` are a special case of ERTP `payments`. They are linked to a specific contract `instance`, and
+having one gives you the right to participate in that contract `instance`, for example, by making offers in it.
+
+There are two main ways for contract users to get an `invitation`:
+- If you create the contract `instance`, you get a special creator `invitation`.
+- Someone who holds the right objects has created an `invitation` for a contract `instance` and given it to
```

Comment:
```suggestion
- Someone (possibly you) who holds the right objects has created an `invitation` for a contract `instance` and gives it to
```

"Someone ... gives it to you" is the subject of "two main ways ... to get an invitation"